### ISSUE TITLE
[BUG] FreeBSD installs deprecated bind911 by default, should install bind916

### ISSUE THREAD

**Issue by darkpixel** [saltstack-formulas/bind-formula#148]
https://github.com/saltstack-formulas/bind-formula/blob/e398d7cbd58b7fce0293e2a2ebe349db4d986bd8/bind/map.jinja#L79

The bind911 package is deprecated and the formula should probably default to the bind916 package.

**Comment by myii** [comment]
@darkpixel Appreciate the report, would you mind preparing a PR, similar to the one that upgraded to `bind911` a couple of years back (#91)?

**Comment by javierbertoli** [comment]
I'm not a FreeBSD user so I don't know the packages naming schema it uses, but is there a _virtual_ with a standarized name that has the 'latest version' as a dependecy?

Ie, `bind9` which installs _'whichever bind9xx version is the current one'_ as a dependency?

**Comment by darkpixel** [comment]
Nope.  You have to manually specify which package version you want.  For example, they have two salt packages: 'py27-salt' and 'py37-salt'.  If you want to upgrade or change versions, you need to explicitly make the change.  You wouldn't want your server automatically changing major or minor package versions on you and breaking something.  (Security updates aren't considered a major or minor version change).

**Comment by javierbertoli** [comment]
oh, well :smile: Then, if you can add those 2 lines @daks suggested in the `pillar.example` and ammend the commit subject, we'll be OK to merge. Thanks!

**Comment by myii** [comment]
> oh, well smile Then, if you can add those 2 lines @daks suggested in the `pillar.example` and ammend the commit subject, we'll be OK to merge. Thanks!

@javierbertoli Was this comment intended for https://github.com/saltstack-formulas/apt-formula/pull/64?

**Comment by javierbertoli** [comment]
Yeah. Sorry for the noise :roll_eyes:
